app/dashboard/views.py

```diff
@@ -4172,3 +4238,158 @@ def save_tribe(request,handle):
                     'is_my_org': False,
                 },
                 status=401)
+
+
+@csrf_exempt
+@require_POST
+@staff_member_required
+def create_bounty(request):
+
+    '''
+        TODO
+        - evaluate validity of duplicate / redundant data in models
+        - wire in email
+        - add activty
+    '''
+    response = {
+        'status': 400,
+        'message': 'error: Bad Request. Unable to create bounty'
+    }
+
+    if not request.user.is_authenticated:
+        response['message'] = 'error: user needs to be authenticated to create bounty'
+        return JsonResponse(response)
+
+    profile = request.user.profile if hasattr(request.user, 'profile') else None
+
+    if not profile:
+        response['message'] = 'error: no matching profile found'
+        return JsonResponse(response)
+
+    if not request.method == 'POST':
+        response['message'] = 'error: create bounty is a POST operation'
+        return JsonResponse(response)
+
+    github_url = request.POST.get("github_url", None)
+    if Bounty.objects.filter(github_url=github_url).exists():
+        response = {
+            'status': 303,
+            'message': 'bounty already exists for this github issue'
+        }
+        return JsonResponse(response)
+
+    bounty = Bounty()
+
+    bounty.bounty_owner_profile = profile
+    bounty.bounty_state = 'open'
+    bounty.title = request.POST.get("title", '')
+    bounty.token_name = request.POST.get("token_name", '')
+    bounty.bounty_type = request.POST.get("bounty_type", '')
+    bounty.project_length = request.POST.get("project_length", '')
+    bounty.estimated_hours = request.POST.get("estimated_hours", None)
```

Comment:
with `.get` if you don't specify the 2nd argument the default is `None` so you don't need to add it

Reply:
Oh  noice :O